### PR TITLE
Fix conversion of TensorData, TensorsData to json

### PR DESCRIPTION
**Pull request by xadupre**
### Description
Fix write_calibration_table to support TensorData, TensorsData

### CODE REVIEW

**Comment by chilo-ms** [comment]
Thanks!
TRT EP actually is reading the calibration.flatbuffers not the pure txt file, so could you please help add the code to block as below?

```python

700     zero = np.array(0)
701     for key in sorted(calibration_cache.keys()):
702         values = calibration_cache[key]
703         d_values = values.to_dict()
704         floats = [
705             float(d_values.get("highest", zero).item()),
706             float(d_values.get("lowest", zero).item()),
707         ]
708         value = str(max(floats))  # str(max(abs(values[0]), abs(values[1])))
709
710         flat_key = builder.CreateString(key)
711         flat_value = builder.CreateString(value)

```

**Comment by xadupre** [comment]
> ```python
> builder.CreateString(key)
> ```

I don't know block very well. Do you know the file I should modify?

**Comment by chilo-ms** [comment]
> > ```python
> > builder.CreateString(key)
> > ```
> 
> I don't know block very well. Do you know the file I should modify?

The onnxruntime/python/tools/quantization/quant_utils.py

> > ```python
> > builder.CreateString(key)
> > ```
> 
> I don't know block very well. Do you know the file I should modify?

It's here https://github.com/xadupre/onnxruntime/blob/qdq_json/onnxruntime/python/tools/quantization/quant_utils.py#L698

**Comment by xadupre** [comment]
> > > ```python
> > > builder.CreateString(key)
> > > ```
> > 
> > 
> > I don't know block very well. Do you know the file I should modify?
> 
> It's here https://github.com/xadupre/onnxruntime/blob/qdq_json/onnxruntime/python/tools/quantization/quant_utils.py#L698

Sorry, I did not see, it was just above. I just pushed the changes.

File: onnxruntime/python/tools/quantization/quant_utils.py [hunk 1]
```diff
@@ -671,21 +671,41 @@ def write_calibration_table(calibration_cache, dir="."):
...
     key_value_list = []
     for key in sorted(calibration_cache.keys()):
         values = calibration_cache[key]
-        value = str(max(abs(values[0]), abs(values[1])))
+        d_values = values.to_dict()
+        floats = [
+            float(d_values.get("highest", zero).item()),
+            float(d_values.get("lowest", zero).item()),
+        ]
+        value = key + " " + str(max(floats))
```
Comment (chilo-ms) [inline onnxruntime/python/tools/quantization/quant_utils.py:708]: please change this line to` value = str(max(floats))`

Comment (xadupre) [reply onnxruntime/python/tools/quantization/quant_utils.py]: done

**Review by chilo-ms — APPROVED** [review]

**Pull request merged**